From d619fb5e1c07f15eb6b9156a389fad85e2924e57 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 19 Aug 2011 08:08:18 +0200
Subject: [PATCH] dracut: unset LD_LIBRARY_PATH

LD_LIBRARY_PATH is not set in the initramfs, so it should not be set
while finding our libraries.
---
 dracut |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/dracut b/dracut
index cf27b23..dfa71a1 100755
--- a/dracut
+++ b/dracut
@@ -265,6 +265,7 @@ fi
 
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
 export PATH
+unset LD_LIBRARY_PATH
 
 [[ $debug ]] && {
     export PS4='${BASH_SOURCE}@${LINENO}(${FUNCNAME[0]}): ';
